nautobot/docs/user-guides/graphql.md

```diff
@@ -225,7 +225,7 @@ query {
 
 Now that we've explored how to use the GraphiQL interface to help us create GraphQL queries, let's take our queries and call them with the REST API. This is where the real advantage is going to come in to play, because it will allow us to utilize these queries in a programmatic way.
 
-<img src="/user-guides/images/graphql/10-graphiql-swagger.png" alt="GraphQL: Swagger" width="400">
+<img src="../images/graphql/10-graphiql-swagger.png" alt="GraphQL: Swagger" width="400">
```

Comment:
For whatever reason this image is not loading.

Reply:
See: https://nautobot--296.org.readthedocs.build/en/296/user-guides/images/graphql/10-graphiql-swagger.png

Reply:
Ugh. Good find. There was a typo in the img url.